plugins/bootc.py

```diff
@@ -112,16 +113,5 @@ def configure(self):
                 self.extargs.append("--sysroot=%s" % self.opts.installroot)
 
     def run(self):
-        # combine stdout and stderr; capture text output
-        proc = subprocess.Popen(
-            self.extargs, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, text=True
-        )
-
-        # stdout and stderr will be combined in stdout, err will be None here
-        (out, err) = proc.communicate()
-
-        if proc.returncode != 0:
-            logger.critical(out)
-            raise dnf.cli.CliError
-        else:
-            logger.info(out)
+        proc = subprocess.Popen(self.extargs, stdout=sys.stdout, stderr=sys.stderr)
```

Comment:
The main reason I did the combining of stdout and stderr and use logger.critical(out) is because I thought we should capture rpm-ostree output and put it in the dnf log.  However, I guess it doesn't really matter since it's ultimately running rpm-ostree and I imagine it has its own logging mechanism.  I am fine with this change, but we should keep in mind that someone may ask down the road where the rpm-ostree stuff is in the dnf logs and if that comes up, we're back in this conversation.

Reply:
Yeah, eventually we should make sure to log to `/var/log/dnf.log`. It's just tricky to do now when all we have is unstructured output from rpm-ostree.